packages/dds/map/src/mapKernel.ts

```diff
@@ -548,6 +548,18 @@ export class MapKernel {
         return previousLocalValue;
     }
 
+    private getClearLocalOpMetadata(op: IMapClearOperation, previousMap?: Map<string, ILocalValue>): unknown {
```

Comment:
rather than returning unknown can we use or define a type?

Reply:
@dannimad, see *LocalOpMetadata types earlier in the file.